### PR TITLE
removes common channel

### PR DESCRIPTION
**Pull request by Iamgoofball**
@hornygranny

### CODE REVIEW

**Comment by Zilenan91** [comment]
Don't tell me he's testmerging this

**Comment by hornygranny** [comment]
:ok_hand:

File: code/modules/flufftext/Hallucination.dm [hunk 1]
```diff
@@ -524,7 +524,7 @@ var/list/non_fakeattack_weapons = list(/obj/item/weapon/gun/projectile, /obj/ite
 		for(var/mob/living/carbon/human/H in living_mob_list)
 			humans += H
 		person = pick(humans)
-		target << target.compose_message(person,person.languages,pick(radio_messages),"1459",person.get_spans())
+		target << target.compose_message(person,person.languages,pick(radio_messages),"1213",person.get_spans())
```
Comment (phil235) [inline code/modules/flufftext/Hallucination.dm]: what channel is 1213?

**Comment by AlexanderUlanH** [comment]
Give assistants service channel if you do this

**Comment by JJRcop** [comment]
So if engineers wanted to, since the headsets still use the channel, they could restore it?

**Comment by Supermichael777** [comment]
OH GOD GOOF THAT WAS A JOKE IM SORRY.

too bad radio headsets cant be receive only on certain chanels.

**Comment by TrustyGun** [comment]
why

**Comment by Thunder12345** [comment]
forwhatpvrpose

**Comment by PKPenguin321** [comment]
While I believe too much ease of communication is a bad thing, this isn't the right way to fix it. :-1:

**Comment by Incoming5643** [comment]
are you trying to kill the server

File: code/game/objects/items/devices/radio/radio.dm [hunk 2]
```diff
@@ -587,3 +590,7 @@
 
 /obj/item/device/radio/off	// Station bounced radios, their only difference is spawning with the speakers off, this was made to help the lag.
 	listening = 0			// And it's nice to have a subtype too for future features.
+
+/obj/item/device/radio/off/New()
+	..()
+	qdel(src)
```
Comment (duncathan) [inline code/game/objects/items/devices/radio/radio.dm]: ???

Comment (Iamgoofball) [inline code/game/objects/items/devices/radio/radio.dm]: common channel eradication

**Comment by KorPhaeron** [comment]
Well that was fun

**Comment by Archie700** [comment]
test ended even hornygranny thought it sucked

**Pull request closed without merging**